cmd/network-console-collector/resources/openshift/deployment.yaml

```diff
@@ -0,0 +1,126 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: network-console-collector
+  labels:
+    app.kubernetes.io/part-of: network-console
+    app.kubernetes.io/name: network-console-collector
+  annotations:
+    serviceaccounts.openshift.io/oauth-redirectreference.primary: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"network-console"}}'
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/part-of: network-console
+    app.kubernetes.io/name: network-console-collector
+  name: network-console-collector
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/part-of: network-console
+      app.kubernetes.io/name: network-console-collector
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/part-of: network-console
+        app.kubernetes.io/name: network-console-collector
+        app.kubernetes.io/component: server
+    spec:
+      serviceAccountName: network-console-collector
+      containers:
+      - name: network-console
+        image: quay.io/ckruse/network-console-collector:latest
+        imagePullPolicy: Always
+        args:
+          - -listen=127.0.0.1:8080
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          runAsNonRoot: true
+        volumeMounts:
+        - mountPath: /etc/messaging/
+          name: skupper-local-client
+        ports:
+        - containerPort: 8080
+          name: web
+          protocol: TCP
+      - name: oauth-proxy
+        image: quay.io/openshift/origin-oauth-proxy:4.14.0
+        imagePullPolicy: Always
+        args:
+        - --https-address=:8443
+        - --provider=openshift
+        - --openshift-service-account=network-console-collector
+        - --upstream=http://127.0.0.1:8080
+        - --tls-cert=/etc/tls/proxy-certs/tls.crt
+        - --tls-key=/etc/tls/proxy-certs/tls.key
+        - --cookie-secret=SECRET
+        - -skip-auth-regex=^/metrics
+        ports:
+        - containerPort: 8443
+          name: https
+          protocol: TCP
+        resources: {}
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          runAsNonRoot: true
+          seccompProfile:
+            type: RuntimeDefault
+        volumeMounts:
+        - mountPath: /etc/tls/proxy-certs/
+          name: network-console-certs
+      volumes:
+      - name: network-console-certs
+        secret:
+          defaultMode: 420
+          secretName: network-console-certs
+      - name: skupper-local-client
+        secret:
+          defaultMode: 420
+          secretName: skupper-local-client
```

Comment:
Depends on the skupper-local-client secret being present - which is thus far only present in skupper 1. That means all of these examples only run alongside v1 for now.

The name of this static target will likely change in v2.